src/hooks/useMovietrailer.ts

```diff
@@ -0,0 +1,40 @@
+import { useEffect } from "react";
+import { useDispatch } from "react-redux";
+import { API_OPTIONS } from "../utils/constants";
+import { addTrailerVideo } from "../utils/moviesSlice";
+
+interface VideoDetails {
+  id: string;
```

Comment:
type instead of interface if not extending

Reply:
resolved